static/js/desktopStatistics.js

```diff
@@ -620,8 +619,24 @@ function buildCharts() {
       title: "VMs"
     }
   );
-  createBarChart('#number-of-cpus', dummyData.cpus.dataset);
-  createBarChart('#size-of-ram', dummyData.ram.dataset);
+  createBarChart('#number-of-cpus', dummyData.cpus.dataset, {
+    colors: ['#ed764d', '#925375'],
```

Comment:
Can you use '#E95420', '#772953' instead.

Reply:
@anthonydillon just spotted that too 👍